lib/ansible/modules/system/ufw.py

```diff
@@ -471,8 +471,13 @@ def ufw_version():
                     current_default_values["incoming"] = extract.group(1)
                     current_default_values["outgoing"] = extract.group(2)
                     current_default_values["routed"] = extract.group(3)
-                    if current_default_values[params['direction']] != value:
-                        changed = True
+                    if params['direction'] is None:
+                        if any(v not in (value, 'disabled') for v in current_default_values.values()):
```

Comment:
I'm not a big fan with this... there's too much instructions on the same line. It's not easy to understand for other contributers.

Reply:
I've expanded it into a `for` loop.